src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -80,7 +80,7 @@ internal bool IsReplaying
                 this.isReplaying = value;
             }
         }
-
```

Comment:
nit: we can probably undo this whitespace change.

Reply:
Reverted